stimpacks/kubernetes/configure.go

```diff
@@ -29,9 +33,13 @@ func (k *Kubernetes) configureContext() error {
 	if sa == "" {
 		sa = k.stim.ConfigGetString("kube-service-account") //TODO: depreciated config should be removed
 	}
-	saFilter := k.stim.ConfigGetString("kube.config.serviceaccountfilter")
+	saFilter := k.stim.ConfigGetString("kube.service-account.filter")
```

Comment:
Whats the justification for renaming this?  Also you did not document this change, i would break anyone currently using it.

Reply:
I do prefer to keep these under the command/subcommand they are used with, since this applies to `stim kube config`  putting it under`kube.config.###` make more sense and is easier to understand.